### DISCUSSION TITLE
Skip all VMs

### DISCUSSION THREAD

**Discussion by vdudejon** [bb-Ricardo/netbox-sync#67] [Q&A] [answered]
Is it possible to skip importing all VMs for a vCenter?  I tried vm_exclude_filter = * but it dodn't seem to like that.

**Comment by bb-Ricardo** [comment, marked as answer]
Hi,

Yeas that's possible. You had it almost right. Because these are regular expressions you need to add a dot and then it would look like this:
```
vm_exclude_filter = .*
```
This should exclude all VMs.

**Comment by vdudejon** [comment]
Thanks!  That did it.
